ra-trustvox/components/TrustvoxProductDetailsRate.tsx

```diff
@@ -0,0 +1,50 @@
+interface Props {
+  /**
+   * @ignore
+   */
+  productId: string;
+}
+
+export default function TrustvoxProductDetailsRate({ productId }: Props) {
+  return (
+    <>
+      <style type="text/css">
+        {`
+          .trustvox-widget-rating,
+          .trustvox-widget-rating .ts-shelf-container,
+          .trustvox-widget-rating .trustvox-shelf-container,
+          .trustvox-widget-rating .ts-rate {
+            display: flex;
+            align-items: center;
+          }
+
+          .trustvox-widget-rating span.rating-click-here {
+            display: flex;
+            color: #DAA81D;
+          }
+
+          .trustvox-widget-rating:hover span.rating-click-here {
+            text-decoration: underline;
+          }
+
+          .ts-shelf-container .ts-shelf-rate {
+            top: 0 !important;
+          }
+        `}
+      </style>
```

Comment:
Question @guitavano should we use tailwind here? even outside a site?

Reply:
Hey @guitavano, can you help clarify this?

Reply:
It is ok to not have tailwind here.